### PR TITLE
feature/traplog-remove

### LINKED ISSUE #367: kill off traplog
The traplog table is completely useless.

Let remove it.

### PR DESCRIPTION
**Pull request by jrouzierinverse**
# Description

Remove the traplog table
# Impacts

pfsetvlan
# Issue

Fixes #367

# NEWS file entries

# Delete branch after merge

NO

# Upgrade file entries

The traplog table is now deprecated.
If wish to reclaim the space in the database.
The table should be manually removed.

### CODE REVIEW

File: lib/pf/traplog.pm [hunk 1]
```diff
@@ -1,378 +0,0 @@
...
-=head1 DESCRIPTION
-
-pf::traplog contains the functions necessary to read and manage the SNMP
-traps history gathered by PacketFence from the switches on the network.
-
-=cut
-
-use strict;
-use warnings;
-use RRDs;
```
Comment (fdurand) [inline lib/pf/traplog.pm:18]: MAybe we can remove this dep

**Comment by fdurand** [comment]
upgrade-x-y-z -> drop table traplog;

**Comment by jrouzierinverse** [comment]
I thought that may not be the best thing to do.

I was going to suggest to add a note in the upgrade guide about removing it.

James Rouzier
jrouzier@inverse.ca :: +1.514.447.4918 (x115)  ::  http://www.inverse.ca
Inverse inc. :: Leaders behind SOGo (http://www.sogo.nu) and PacketFence (http://www.packetfence.org)

**Comment by julsemaan** [comment]
I think I'd agree with James on the drop. Some old mature environments might want to keep it. 

If we do drop it in the upgrade script, we should add a note in the upgrade guide about it so people can backup the table before it gets deleted

**Comment by fdurand** [comment]
Rebase please

**Comment by jrouzierinverse** [comment]
Done

**Comment by louismunro** [comment]
Please add the note to the UPGRADE about this being deprecated.
That seems to be all that's missing to merge.

**Comment by jrouzierinverse** [comment]
Done

**Comment by louismunro** [comment]
I believe you must push the commit too...
😁

**Comment by jrouzierinverse** [comment]
Sorry which direction are we going.

* Deleting the traplog warning the users that they should back it up.
* Let it be and let the user remove it manually.

**Comment by jrouzierinverse** [comment]
Done

**Pull request merged**